estate_account/models/estate_property.py

```diff
@@ -0,0 +1,25 @@
+from odoo import models, Command
+
+
+class EstateAccountProperty(models.Model):
+    _inherit = "estate.property"
+
+    def action_sell_offer(self):
+        for record in self:
+            account_dictionnary = {"move_type": "out_invoice",
+                                   "journal_id": self.env["account.journal"].search([('type', '=', 'sale')]).id,
+                                   "partner_id": record.buyer_id.id,
+                                   "line_ids": [
+                                       Command.create({
+                                           "name":"6% of the property price",
+                                           "quantity":"1",
+                                           "price_unit":record.selling_price*0.06
+                                       }),
+                                       Command.create({
+                                           "name":"Administrative fees",
+                                           "quantity":"1",
+                                           "price_unit":100.00
+                                       })
+                                   ]} 
```

Comment:
Obviously, it's a dictionary... :) (And only one 'n' in dictionary)
Maybe prettier like that.

```suggestion
            account_move = {
                "move_type": "out_invoice",
                "journal_id": self.env["account.journal"].search([('type', '=', 'sale')]).id,
                "partner_id": record.buyer_id.id,
                "line_ids": [
                      Command.create({
                               "name":"6% of the property price",
                               "quantity":"1",
                               "price_unit":record.selling_price*0.06
                      }),
                      Command.create({
                               "name":"Administrative fees",
                               "quantity":"1",
                               "price_unit":100.00
                      })
                ]} 
```